src/SwipeButton.tsx

```diff
@@ -15,11 +24,11 @@ interface SwipeButtonProps extends SwipeButtonPropsExtends {
   onComplete: () => void;
 
   /**
-   * The with of the button
-   * 
-   * @default 90% of the screen width
+   * The width of the button
+   *
+   * @default 100% of the screen width
    */
-  width?: number;
+  width?: string;
```

Comment:
I don't think changing the type of this is the right way to go. Changing this will result in a typescript error for everyone that has this value parsed as a `number`. What about changing this to be either a `string` or a `number`

Reply:
Good point. However, this would only impact users who update the package version correct? Nonetheless, I think you're right and it does make sense to accept both strings and numbers.